tests/functional/UseCase/Workspace/ResetWorkspacePasswordTest.php

```diff
@@ -62,7 +62,6 @@ public function testResetWorkspacePassword(): void
             $this->fail('Should fail');
         } catch (Throwable $e) {
             $this->assertInstanceOf(DriverException::class, $e);
-            $this->assertSame(210, $e->getCode());
```

Comment:
No, podle tohohle kodu funguje overovani v SAPI testech, takze by to tu melo zustat.

Reply:
No akorát sa to mění stím PDO->ODBC přechodem jestli sa ti to chce debugovat kde je problém tak na hoď oko v utils. Já bych to teď neřešil a změnil ten test v klientovi.

Reply:
OK, zkusim to pripadne tam...